src/status_im/chat/commands/impl/transactions/styles.cljs

```diff
@@ -71,8 +71,9 @@
 
 (defn command-send-amount-text
   [outgoing]
-  {:font-size 22
-   :color     (if outgoing colors/white colors/blue)})
+  {:font-size   22
+   :font-weight 600
```

Comment:
is it possible to use both number and strings?

Reply:
the react-native property "fontWeight" needs to be a string on react-native, however this is used to build a "fontFamily" in the `typography` namespace and is dissoc from the style. I replaced all the fontWeight integer by strings in a new commit that is coming soon